services/user.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> get_user_model():
```

Comment:
The return type annotation should specify the actual user model class. Consider using `User` or `get_user_model()` to get the class and then use it for type annotation, e.g., `-> Type[User]`.